src/main/common/filter.c

```diff
@@ -65,6 +65,63 @@ FAST_CODE float pt1FilterApply(pt1Filter_t *filter, float input)
     return filter->state;
 }
 
+// PT2 Low Pass filter
+
+float pt2FilterGain(float f_cut, float dT)
+{
+    float RC = 1 / ( 3.10f * M_PIf * f_cut); // cutoff 1.55x compared to PT1
```

Comment:
Maybe keep the 2pi for clarity
```float RC = 1 / ( 2.0f * M_PIf * f_cut * 1.55f); // cutoff 1.55x compared to PT1```

Reply:
This is OK for rc smoothing since the filter is only updated infrequently.
However if say PT2 code was to replace biquad in a dynamic lowpass, and in fact even for PT1, the k calculation gets called every PID loop.
It's always troubled me that most of the k calculation is repeated every time, rather than having a pre-calculated constant for the constant bits. 
Maybe we can pre-calculate `2.0f * M_PIf * 1.55f` or just put that into a static defined value somewhere?

Reply:
Take into account that the compiler is more or less "smart". It usually detects when the numbers are fixed, and replaces the multiplication by the result of them. So at most places is better a "more clear" code than a "faster" one because in the end, both are equal faster.

Reply:
I'm not expert on compilers :-( 
If that's definitely true we can put `M_Pif` back in, but I'd prefer not.
Kluttvott explained that maths for the correction factor is
1 / sqrt( (2^(1 / n) - 1) )
I've provided this in a comment.